src/components/Picker/BasePicker/index.js

```diff
@@ -11,7 +11,7 @@ class BasePicker extends React.Component {
         super(props);
 
         this.state = {
-            selectedValue: this.props.value || this.props.defaultValue,
+            selectedValue: this.props.defaultValue,
```

Comment:
The solution looks good. but now why do we need this state?

Reply:
I asked myself the same question yesterday and did some investigation and I _think_ that's needed because `RNPickerSelect` seems to work as a controlled input only (is this true?) and we need to keep track of the value locally.

Reply:
Hmm `defaultValue` is the prop passed by `Form` though and usages of outside of `Form` use `value` for now, so I think we need both at the moment?

Reply:
Ok. I see. May be leave a comment for that.

Reply:
The problem is that `Form` doesn't provide and update a `value` prop (like ProfilePage.js). `Form` provides `defaultValue` but I think it only serves as first value to fill `Picker` at mount because `Form` doesn't update it (except for inputs with shouldSaveDraft prop). 

Passing `defaultValue` you are telling `Picker` to be a controlled component expecting `defaultValue` to be updated so I added `selectValue` state for `Picker` could keep track of itself on absence  of parent component that could update its value but only providing a first value like `defaultValue`.